packages/client/components/MenuItemWithIcon.tsx

```diff
@@ -3,40 +3,32 @@ import React from 'react'
 import {PALETTE} from '~/styles/paletteV3'
 import {ICON_SIZE} from '~/styles/typographyV2'
 import Icon from './Icon'
-
-const Wrapper = styled('div')({
-  alignItems: 'center',
-  display: 'flex',
-  paddingLeft: 8,
-  paddingRight: 8,
-  width: '100%'
-})
+import MenuItemLabel from './MenuItemLabel'
 
 const ItemIcon = styled(Icon)({
   color: PALETTE.SLATE_600,
   fontSize: ICON_SIZE.MD24,
-  padding: 8
+  marginRight: 8
 })
 
-const Label = styled('div')({
-  color: PALETTE.SLATE_700,
-  fontSize: 14,
-  lineHeight: '32px'
+const IconWrapper = styled('span')({
+  opacity: 0.7,
+  height: ICON_SIZE.MD24,
+  marginRight: 8
```

Comment:
Why do the opacity & margins in these two styled-components need to be updated? Would you mind giving me an example use case / screenshot? We generally try to use padding wherever possible rather than margin as margin can throw off other components' positioning.

Also, when components like this are being reused throughout the app, I'm also hesitant to update them unless there's no other option as we'll need to check the UI at different breakpoints in each of the components that it's being used.